### PR TITLE
Feature/confit and table support in markdown

### PR DESCRIPTION
**Pull request by uglow**

### CODE REVIEW

**Comment by uglow** [comment]
@rodleviton Please review

**Comment by uglow** [comment]
I think the reason the first build passed is due to the caching by Travis of the node_modules. Because I added Node 6 to the build matrix, the node_modules were re-fetched and chalk was not listed as a dependency in package.json.

This kind of error is one of the dangers with caching node modules on Travis.

File: src/tasks/getContent.js [hunk 1]
```diff
@@ -61,13 +64,13 @@ module.exports = (section, meta, callback) => {
   }
 
   function compileSnippet(args) {
-    var hasExtension = path.extname(args[0]);
-    var template = '';
+    //let hasExtension = path.extname(args[0]);   // TODO: Remove?
```
Comment (rodleviton) [inline src/tasks/getContent.js]: Yes this needs to be removed. This is now handled in `getSnippet` which allows a string template to be parsed as a snippet.

**Comment by rodleviton** [comment]
@uglow this is great. Can't thank you enough for your efforts here! 👍 👍 👍

**Pull request merged**